templates/haproxy-reload

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# This script checks whether a reload of haproxy was requested, regenerates its
+# configuration file and reloads it.  The script is run via cron every minute.
+# This ensures that haproxy is reloaded at most once a minute.
+
+set -e
+
+# Name of file that indicates haproxy needs to be reloaded.
+HAPROXY_RELOAD_FILE="{{ LOAD_BALANCER_RELOAD_FILE }}"
+
+if [ -f "$HAPROXY_RELOAD_FILE" -o "$1" = "-f" ]; then
+    rm "$HAPROXY_RELOAD_FILE"
```

Comment:
Thanks, you are right.  I didn't run into this since usually the file will exist in the last step of the role.  Only if the cron job happens to have run earlier it will be already deleted.  The easiest fix is to use

    rm -f "$HAPROXY_RELAOD_FILE"

which won't complain if the file does not exist.

(In theory, the last step of the playbook is redeundant, since the cron job will take care of restarting the service.  However, this seems to implicit to me, and might break silently if the tasks in the role are reordered or refactored.)